pkg/agent/config.go

```diff
@@ -8,18 +8,21 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
-// Config wraps the options for a run of the agent
+// Config wraps the options for a run of the agent.
 type Config struct {
 	Schedule      string         `yaml:"schedule"`
 	Token         string         `yaml:"token"`
-	Endpoint      endpoint       `yaml:"endpoint"`
+	Endpoint      Endpoint       `yaml:"endpoint"`
 	DataGatherers []dataGatherer `yaml:"data-gatherers"`
 }
 
-type endpoint struct {
-	Host string `yaml:"host"`
-	Path string `yaml:"path"`
+// Endpoint is the configuration of the server where to post the data.
+type Endpoint struct {
+	Protocol string `yaml:"protocol"`
```

Comment:
Should protocol default to http?

Reply:
ok, let's make it default to http, since this is going to be used for local development mainly.